.expeditor/macos_universal_package.pipeline.yml

```diff
@@ -1 +1,7 @@
 # This pipeline is currently under development.
+project-name: universal-package
```

Comment:
You may also want to add an adhoc version of this build so that you can test without needing to merge first. See https://expeditor.chef.io/docs/pipelines/omnibus/